File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.java
class Solution {
    public double average(int[] salary) {
        
        Arrays.sort(salary);
        int count = salary.length;
        int sum=0;
        for(int i=1;i<count-1;i++){
            sum+=salary[i];
        }
  
        return (double) sum/(count-2);
    }
}

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.js
/**
 * @param {number[]} salary
 * @return {number}
 */
var average = function(salary) {
    salary.sort((a,b)=>a-b)
    let count = salary.length
    let sum=0
    for(let i=1;i<count-1;i++){
        sum+=salary[i]
    }
    return sum/(count-2)
};